backend/src/main.py

```diff
@@ -43,3 +48,10 @@ def initialize_backend_application() -> fastapi.FastAPI:
         workers=settings.SERVER_WORKERS,
         log_level=settings.LOGGING_LEVEL,
     )
+new_loop = asyncio.new_event_loop()
```

Comment:
We should use a better way to do this. We need to make sure "__main__" function only start up the fastAPI framework.